tests/PHPStan/Rules/Properties/data/bug-6398.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Bug6398;
```

Comment:
Does this fix also https://github.com/phpstan/phpstan/issues/9239?

Reply:
Don't know, most likely not. If it gets merged and if it does, we should notice I guess. But many ifs in that sentence 😊